### PR TITLE
WebSocket.send_bytes is a coroutine as of aiohttp 3.0 changing transp…

### LINKED ISSUE #15: coroutine never awaited
```
aiogremlin/driver/aiohttp/transport.py:22: RuntimeWarning: coroutine 'ClientWebSocketResponse.send_bytes' was never awaited
  self._ws.send_bytes(message)
```

I'm getting this warning while using goblin. It also seems that my db is not receiving the request. I added a return to that line and both problems went away.

aiogremlin 3.2.6
goblin 2.1.0

### PR DESCRIPTION
**Pull request by xzy3**
…ort.write to await the coroutine if necessary.

Fixes #15

### CODE REVIEW

File: aiogremlin/driver/aiohttp/transport.py [hunk 1]
```diff
@@ -18,8 +19,10 @@ def __init__(self, loop):
         self._ws = await self._client_session.ws_connect(url)
         self._connected = True
 
-    def write(self, message):
-        self._ws.send_bytes(message)
+    async def write(self, message):
+        corou = self._ws.send_bytes(message)
+        if asyncio.iscoroutine(corou):
+          await corou
```
Comment (davebshow) [inline aiogremlin/driver/aiohttp/transport.py]: Could we use `coro` instead of `corou`? No big deal really

**Comment by davebshow** [comment]
Other than my small inline comment this is LGTM Please update and I will merge. Thanks @xzy3 !

**Comment by xzy3** [comment]
@davebshow Ok I rebased in the variable name change as requested.

**Comment by xzy3** [comment]
A follow up: looking through the history of aiohttp it looks like the `tp`->`type` refactor took place pre 1.0.0 so these changes should be backwards compatible at least to that version. The alias was left in until the 3.0.0 release.

**Comment by mildewey** [comment]
This issue is a blocker. The most basic tutorial in the documentation does not work until this fix goes live. Can we have a new release? 

I'm open to suggestions regarding workarounds if not.

**Pull request merged**